roofit/roofit/inc/RooBernstein.h

```diff
@@ -33,11 +33,13 @@ class RooBernstein : public RooAbsPdf {
 
   Int_t getAnalyticalIntegral(RooArgSet& allVars, RooArgSet& analVars, const char* rangeName=0) const ;
   Double_t analyticalIntegral(Int_t code, const char* rangeName=0) const ;
+  virtual void selectNormalizationRange(const char* rangeName=0, Bool_t force=kFALSE) ;
 
 private:
 
   RooRealProxy _x;
   RooListProxy _coefList ;
+  const TNamed* _refRangeName ;
```

Comment:
No need to put a pointer, just use simple string.
```suggestion
 std::string _refRangeName ;
```